### ISSUE TITLE
[Support] install does script not working

### ISSUE THREAD

**Issue by moebi09** [pivpn/pivpn#1264]
## In raising this issue, I confirm the following:

`{please fill the checkboxes, e.g: [X]}`

- [x] I have read and understood the [contributors guide](https://github.com/pivpn/pivpn/blob/master/CONTRIBUTING.md).
- [x] The issue I am reporting can be *replicated*.
- [x] The issue I am reporting can be *is* directly related to the pivpn installer script.
- [x] The issue I am reporting isn't a duplicate (see [FAQs](https://github.com/pivpn/pivpn/wiki/FAQ), [closed issues](https://github.com/pivpn/pivpn/issues?q=is%3Aissue+sort%3Aupdated-desc+is%3Aclosed), and [open issues](https://github.com/pivpn/pivpn/issues?q=is%3Aissue+sort%3Aupdated-desc+is%3Aopen)).

pi@raspberryvpn:~ $ curl -L https://install.pivpn.io | bash
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   162  100   162    0     0    299      0 --:--:-- --:--:-- --:--:--   299
100 83358  100 83358    0     0   103k      0 --:--:-- --:--:-- --:--:--  103k
:::
::: sudo will be used for the install.
::: Hostname length OK
::: Verifying free disk space...
:::
::: Package Cache update is needed, running apt-get update -y ...
 done!
:::
::: Checking apt-get for upgraded packages.... done!
:::
::: There are 4 updates available for your system!
::: We recommend you update your OS after installing PiVPN! 
:::
:::    Checking for git... already installed!
:::    Checking for tar... already installed!
:::    Checking for wget... already installed!
:::    Checking for curl... already installed!
:::    Checking for grep... already installed!
:::    Checking for dnsutils... already installed!
:::    Checking for whiptail... already installed!
:::    Checking for net-tools... already installed!
:::    Checking for bsdmainutils... already installed!
:::    Checking for iptables-persistent... already installed!
::: Using User: pi
:::
::: Checking for existing base files...
:::    Checking /usr/local/src/pivpn is a repo... OK!
:::     Updating repo in /usr/local/src/pivpn... done!
main: line 1140: [: : integer expression expected
main: line 1152: [: : integer expression expected
::: Install Complete...
::: Restarting services...
:::    Checking for unattended-upgrades... already installed!
::: Setupfiles copied to /etc/pivpn//setupVars.conf
::: Installing scripts to /opt/pivpn...
main: line 2288: /etc/bash_completion.d/pivpn: No such file or directory
 done.
::: Flushing writes to disk...
::: done.
:::

### Issue
Installscript started.
At first not really seen there is a fault.

Try d "pivpn add" this didn't work. Then retried different scripts. Tried "su" for root rights. Nothing changes.

Everytime "no such file...."

System DELL office pc x86 64bit 
Intel(R) Core(TM)2 Duo CPU     E7200  @ 2.53GHz
4Gb RAM
Debian GNU/Linux 10 (buster)

### Have you searched for similar issues and solutions?
    (yes/no / which issues?)
Issues 173 and 399 have quite similar problems/output but solution (moving git) does not work...

### Console output of      `curl -L install.pivpn.io | bash`
```
  Output Here it's above (am I wrong?)
```

### Console output of      `pivpn add` or `pivpn add nopass`
```
  Output Here - I am not at this point
```

### Console output of      `pivpn debug`
```
  Output Here
```
### Have you taken any steps towards solving your issue?
```
  which?
```
Google, a lot of "manuals" and knowledge pages read. Reinstall etc etc

**Comment by orazioedoardo** [comment]
Can you try the testing branch? https://github.com/pivpn/pivpn/wiki#testing

**Comment by 4s3ti** [comment]
@orazioedoardo  github wiki should be deprecated, as only a few can contribute to it, use the docs instead pls, https://docs.pivpn.io/install/

**Comment by thedigi321** [comment]
> @orazioedoardo github wiki should be deprecated, as only a few can contribute to it, use the docs instead pls, https://docs.pivpn.io/install/

Hi experienced same issue as moebi09 same errors and everything. I believe there is issues in pivpn repos specially on line 1140 and 1152 both expected integer expressions. moving to testing branch and it did fix my issues.

**Comment by MTB-Joe** [comment]
Dear guys
maybe this helps in the case that the test branch may not work for / be wished by anybody concerned:
I ran into the same issue and debugged it by my own because of assuming the mistake in my own setup:

So far the variable "WIREGUARD_SUPPORT" is just set to "1" if confirmed but never set to "0" if not. So the if-condition in the given lines 1140 / 1152 failed to be checked because "WIREGUARD_SUPPORT" is NULL (empty) instead of "0" if not supported. 

As seen meanwhile this has already been fixed in the test branch (better would have looked there before =) ) - in the current test branch it's in line 466.
